test/integration/containerd_test.go

```diff
@@ -0,0 +1,82 @@
+//go:build integration
+// +build integration
+
+package integration
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/aquasecurity/fanal/image/daemon"
+	"github.com/containerd/containerd"
+	"github.com/containerd/containerd/namespaces"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/stretchr/testify/require"
+	testcontainers "github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	redisImageName  = "index.docker.io/library/redis:latest"
+	mysqlImageName  = "index.docker.io/library/mysql:latest"
+	pythonImageName = "index.docker.io/library/python:3.4-alpine"
+)
+
+func TestGetLocalContainerdImage(t *testing.T) {
+	ctx := context.Background()
+
+	ctx = namespaces.WithNamespace(ctx, "default")
+	normalPath, err := filepath.Abs(".")
+	require.NoError(t, err)
+	hostPath := filepath.Join(normalPath, "data", "test")
+	targetPath := filepath.Join(hostPath, "containerd")
+	defer os.RemoveAll(hostPath)
+	err = os.MkdirAll(targetPath, os.ModePerm)
+	require.NoError(t, err)
+
+	req := testcontainers.ContainerRequest{
+		Name:       "containerd",
+		Image:      "docker.io/geyingqi/dockercontainerd:alpine",
```

Comment:
We should not depend on the 3rd-party image. What if we put Dockerfile in this repository and push the image to ghcr.io/aquasecurity?

Reply:
If you will upload Dockerfile, I will build an image and push it to GHCR. Thanks.